Hi @velo, thank you for the PR.  I would be happy to release this patch.

Please fix the checkstyle error, and ensure that `mvn clean verify` completes successfuly:
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  3.889 s
[INFO] Finished at: 2020-06-30T14:17:29+09:30
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (attach-javadoc) on project kafka-lag-based-assignor: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - kafka-lag-based-assignor-velo/src/main/java/com/github/grantneale/kafka/LagBasedPartitionAssignor.java:79: error: reference not found
[ERROR]  * @see PartitionAssignor
[ERROR]         ^
[ERROR]
[ERROR] Command line was:javadoc @options @packages
[ERROR]
[ERROR] Refer to the generated Javadoc files in 'kafka-lag-based-assignor-velo/target/apidocs' dir.
[ERROR]
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/src/main/java/com/github/grantneale/kafka/LagBasedPartitionAssignor.java
+++ b/src/main/java/com/github/grantneale/kafka/LagBasedPartitionAssignor.java
@@ -76,7 +76,7 @@ import org.slf4j.LoggerFactory;
  * C0: 160,000
  * C1:  50,000
  *
- * @see PartitionAssignor
+ * @see ConsumerPartitionAssignor
  * @see org.apache.kafka.clients.consumer.internals.AbstractPartitionAssignor
  * @see org.apache.kafka.clients.consumer.RangeAssignor
  */

